### DISCUSSION TITLE
Customization for Generation Expension Planning

### DISCUSSION THREAD

**Discussion by Erdemhan** [anyoptimization/pymoo#452] [Q&A]
Hi , 
First of all sorry if it is already answered but couldn't find any information about that.

I want to implement GA with pymoo to optimize GEP problem for that i need vectoral solutions with 352 genes (parameters) and first 176 parameter must be float and next 176 parameter must be integer. Is it possible to implement such individuals? if yes how ?

And if yes I have upper bound limitations for every parameter in second 176 is it possible to implement these upper bounds?

as last Is there a previous gep implementation that I can take as an example?

**Comment by blankjul** [comment]
You can either use the mixed variable implementation shown here: https://pymoo.org/customization/custom.html

Or you directly customize your own GA entirely: https://pymoo.org/customization/custom.html
